test/testdata

```diff
@@ -133,7 +133,8 @@ Scan      Jaguar       JaguarScanTest       basicJaguar8.3        dvb_scan.out
 
 SP        ADF         ADFSPTest             basicADF2007.01       dvb_sp_b.adfout
 SP        ADF         ADFSPTest             basicADF2013.01       dvb_sp_b.adfout
-SP        DALTON      DALTONSPTest          basicDALTON-2013      b3lyp_energy_dvb_sp.out
+SP        DALTON      DALTONSPTest          basicDALTON-2013      sp_b3lyp_dvb.out
+SP        DALTON      DALTONSPTest          basicDALTON-2013      sp_rhf_dvb.out
```

Comment:
Now I understand. I think the current filenames are fine, although there could be minor changes to somewhat better match the NWChem files (i.e. b3lyp -> ks and rhf -> hf).